eng/Versions.props

```diff
@@ -12,6 +12,7 @@
   <PropertyGroup>
     <!-- Match the first three version numbers and append 00 -->
     <DotNetPreviewVersionBand Condition=" '$(DotNetPreviewVersionBand)' == '' ">$([System.Text.RegularExpressions.Regex]::Match($(MicrosoftDotnetSdkInternalPackageVersion), `^\d+\.\d+\.\d`))00</DotNetPreviewVersionBand>
+    <DotNetSdkManifestsFolder>$(DotNetPreviewVersionBand)$([System.Text.RegularExpressions.Regex]::Match($(MicrosoftDotnetSdkInternalPackageVersion), `\-preview.\d+`))</DotNetSdkManifestsFolder>
```

Comment:
So this expression is meant to match, `7.0.100-preview.4.22219.19`, and we want this to come up with `7.0.100-preview.4`.

What happens in `main` where we have `6.0.300-rtm.22214.7`? Should we fallback to `$(DotNetPreviewVersionBand)` if the Regex doesn't match?

Reply:
Let me test this part, I'll add the fallback to fix this up as needed.

Reply:
I think this works fine as is, if there is no `-preview.` string in `$(MicrosoftDotnetSdkInternalPackageVersion)` the match will be empty and `$(DotNetSdkManifestsFolder)` will just be `$(DotNetPreviewVersionBand)`.